radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -1153,7 +1155,9 @@ public void stageWorkingArea() throws StageActionException {
 		public void stage(Action action) throws StageActionException {
 			BiFunction<Action, Stream<Particle>, List<ParticleGroup>> statefulMapper = actionMappers.get(action.getClass());
 			if (statefulMapper == null) {
-				throw new IllegalArgumentException("Unknown action class: " + action.getClass() + ". Available: " + actionMappers.keySet());
+				throw new IllegalArgumentException(
+						"Unknown action class: " + action.getClass() + ". Available: " + actionMappers.keySet()
```

Comment:
In my opinion, it's cleaner to use String.format for longer formatted strings since it makes the intended output more obvious and condensed. Same goes for all other exception strings.

Reply:
Sure, can fix some, but not really related to this PR.

Reply:
True, but since it's already being changed anyway :)